packages/go/ein/ad.go

```diff
@@ -342,6 +342,18 @@ func ParseComputerMiscData(computer Computer) []IngestibleRelationship {
 		}
 	}
 
+	isDC, ok := computer.Properties["isdc"]
```

Comment:
i saw that Rohan added some properties to the Computer struct in his PR for the DCFor edge that might be desired here. here is a link to the change he made

https://github.com/SpecterOps/BloodHound/pull/315/files#diff-f53a6afb92ee71372d78b8ecd56e2f482f9815c46c36457e8adaceb15eda0faaR262

Reply:
Yea, this was a temporary made I change in order to test the esc6a post processing locally. I intend to replace this block of code with the changes in the PR you reference.